hirola/front/templates/front/phone_profile.html

```diff
@@ -135,12 +135,13 @@ <h5 id="price" class="green-text">{{ phone.currency }} {{ phone.price|intcomma }
                                 <div class="col s12">
                                         <div class="row">
                                             <div class="input-field col s12">
-                                                <button class="btn waves-effect waves-light green" id="add-to-cart-button" type="button">Buy Now</button>
+                                                <button class="btn waves-effect waves-light green" id="add-to-cart-button" name="buy_now" type="submit" value="1">Buy Now
+                                                </button>
                                             </div>
                                         </div>
                                         <div class="row">
                                             <div class="input-field col s12">
-                                                <button class="btn waves-effect waves-light green darken-2" id="add-to-cart-button2" type="submit" name="action">Add to Cart
+                                                <button class="btn waves-effect waves-light green darken-2" id="add-to-cart-button2" type="submit" name="action" value="1">Add to Cart
```

Comment:
Hey, I am not sure what the `value="1"` adds to the functionality of the Add to Cart button. I think it will still function without it.

Reply:
for this it just a precaution